.circleci/config.yml

```diff
@@ -1,39 +1,80 @@
 version: 2
+
 jobs:
-  qa:
+
+  build:
     docker:
-    - image: circleci/python:3.6-node
-    working_directory: ~/repo
+      - image: google/cloud-sdk:alpine
+    environment:
+      GCP_PROJECT: scaife-viewer
+      IMAGE_NAME: gcr.io/scaife-viewer/scaife-viewer-build
     steps:
-    - checkout
-    - run:
-        name: Install Python dependencies
-        command: pipenv install --dev
-    - run:
-        name: Run isort
-        command: pipenv run isort -c **/*.py
-    - run:
-        name: Run flake8
-        command: pipenv run flake8 scaife_viewer
-    - run:
-        name: Install static dependencies
-        command: npm ci
-    - run:
-        name: Run eslint
-        command: npm run lint
-    - run:
-        name: Run jest tests
-        command: npm run unit
-    - run:
-        name: Run webpack build
-        command: npm run build
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install Docker Compose
+          command: |
+            apk add --no-cache 'py2-pip' docker
+            pip install 'docker-compose==1.23.2'
+      # - run:
```

Comment:
See my overall review comment, but if we're not enabling these "steps", can we please remove them (and grab 'em from VCS history) rather than landing commented out code?